### PR TITLE
{bio}[GCCcore/13.2.0] nf-core v2.13.1, Markdown v3.6, prompt-toolkit v3.0.36, ...

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

Requires
- [x] #20240

### CODE REVIEW

**Comment by zao** [comment]
I can't see the GitPython/3.1.42 dependency anywhere in develop.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
n1441 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/f4b086487b0f1b7f53a646aaf256a9c5 for a full test report.

**Comment by Flamefire** [comment]
> I can't see the GitPython/3.1.42 dependency anywhere in develop.

Missed it while uploading. Thanks!

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
n1265 - Linux RHEL 8.7 (Ootpa), x86_64, Intel(R) Xeon(R) Platinum 8470 (icelake), Python 3.8.13
See https://gist.github.com/Flamefire/9339630e4f1d1d9cc93163171d7afe9f for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

File: easybuild/easyconfigs/m/Markdown/Markdown-3.6-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+description = """This is a Python implementation of John Gruber's Markdown.
+It is almost completely compliant with the reference implementation, though there are a few known issues.
+Additional features are supported by the Available Extensions.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['ed4f41f6daecbeeb96e576ce414c41d2d876daa9a16cb35fa8ed8c2ddfad0224']
+
```
Comment (Micket) [inline easybuild/easyconfigs/m/Markdown/Markdown-3.6-GCCcore-13.2.0.eb:16]:
```suggestion

builddependencies = [
    ('binutils', '2.40'),
]

```

**Review by Micket — DISMISSED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 8 out of 8 (6 easyconfigs in total)
alvis-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/81b476292ff982c5628c958e50ea8945 for a full test report.

File: easybuild/easyconfigs/p/pytest-workflow/pytest-workflow-2.1.0-GCCcore-13.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+builddependencies = [('binutils', '2.40')]
+dependencies = [
+    ('Python', '3.11.5'),
+    ('PyYAML', '6.0.1'),
+    ('python-isal', '1.6.1'),
+]
+
+download_dep_fail = True
```
Comment (Micket) [inline easybuild/easyconfigs/p/pytest-workflow/pytest-workflow-2.1.0-GCCcore-13.2.0.eb:24]:
```suggestion
```

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Review by Micket — APPROVED** [review]
lgtm again

**Comment by Micket** [comment]
Going in, thanks @Flamefire!

File: easybuild/easyconfigs/m/Markdown/Markdown-3.6-GCCcore-13.2.0.eb (hunk 1 above)
Comment (Flamefire) [reply easybuild/easyconfigs/m/Markdown/Markdown-3.6-GCCcore-13.2.0.eb:16]: IMO Such things should be on a single line to reduce noise and IIRC we do that in most other places. I.e.:
`builddependencies = [('binutils', '2.40')]`

**Comment by Flamefire** [comment]
@Micket #19107 is the PR of this for 12.2. Any chance you can take a look at that too please? Had to update it multiple times already to keep up with other changes

**Pull request merged**